en/flight_controller/kakuteh7v2.md

```diff
@@ -0,0 +1,141 @@
+- Dimensions: 35x35mm
+- Weight: 8g
+
+
+## Where to Buy
+
+The board can be bought from one of the following shops (for example):
+- [Holybro](https://shop.holybro.com/kakute-h7-v2_p1344.html)
+
+:::tip
+The *Kakute H7v2* is designed to work with the *Tekko32* 4-in-1 ESC and they can be bought in combination.
+:::
+
+## Connectors and Pins
+
+| Pin              | Function                              | PX4 default |
+| ---------------- | ------------------------------------- | ----------- |
+| B+ | Battery positive voltage (2S-8S) ||
+| SDA, SCL | I2C connection (for peripherals) ||
+| 5V | 5V output (2A max) ||
+| 3V3 | 3.3V output (0.25A max) ||
+| VI | Video input from FPV camera ||
+| VO | Video output to video transmitter ||
+| CAM | To camera OSD control ||
+| G or GND | Ground ||
+| RSI | Analog RSSI (0-3.3V) input from receiver ||
+| R1, T1 | UART1 RX and TX | TELEM1 |
+| R3, T3 | UART3 RX and TX | NuttX debug console |
+| R4, T4 | UART4 RX and TX | GPS1 |
+| R6, T6 | UART6 RX and TX (R6 also located in the GH plug)| RC port |
+| R7 | UART7 RX (RX is located in the plugs for use with 4-in-1 ESCs) |DShot telemetry|
+| LED | WS2182 addressable LED signal wire (not tested) ||
+| Z- | Piezo buzzer negative leg (Connect buzzer positive leg to 5V pad) ||
+| M1 to M4 | Motor signal outputs (located in plug for use in 4-in-1 ESCs) ||
+| M5 to M8 | Motor signal outputs (located in plug for use in 4-in-1 ESCs) ||
+| Boot | Bootloader button | |
+
+
+<a id="bootloader"></a>
+## PX4 Bootloader Update
+
+TODO: hex file
+
+The board comes pre-installed with [Betaflight](https://github.com/betaflight/betaflight/wiki).
+Before the PX4 firmware can be installed, the *PX4 bootloader* must be flashed.
+Download the [holybro_kakuteh7v2_bootloader.hex](TODO) bootloader binary and read [this page](../advanced_config/bootloader_update_from_betaflight.md) for flashing instructions.
+
+## Building Firmware
+
+To [build PX4](../dev_setup/building_px4.md) for this target:
+
+```
+make holybro_kakuteh7v2_default
+```
+
+## Installing PX4 Firmware
+
+The firmware can be installed in any of the normal ways:
+
+- Build and upload the source:
+
+  ```
+  make holybro_kakuteh7v2_default upload
+  ```
+- [Load the firmware](../config/firmware.md) using *QGroundControl*.
+  You can use either pre-built firmware or your own custom firmware.
+
+:::note
+KakuteH7v2 is supported with PX4 main and v1.14 or newer.
+:::
+
+## PX4 Configuration
+
+In addition to the [basic configuration](../config/README.md), the following parameters are important:
+
+Parameter | Setting
+--- | ---
+[SYS_HAS_MAG](../advanced_config/parameter_reference.md#SYS_HAS_MAG) | This should be disabled since the board does not have an internal mag. You can enable it if you attach an external mag.
+[MOT_ORDERING](../advanced_config/parameter_reference.md#MOT_ORDERING) | If you use a 4-in-1 ESC with Betaflight motor assignment, this parameter can be set accordingly.
```

Comment:
Actuator tuning is part of basic configuration, so I am just deleting this parameter.
```suggestion
```